src/store.rs

```diff
@@ -1,11 +1,11 @@
-use itertools::Itertools;
+use generic_array::typenum::{U4, U6, U8};
```

Comment:
Do we want to use const_generics here instead of `typenum`?

Reply:
This flows from Neptune. Switching to const generics there seems to require a backward step in terms of usability (despite also providing benefits), so I haven't changed anything (yet at least).